packages/web3torrent/src/contexts/initialization-context.ts

```diff
@@ -0,0 +1,26 @@
+import {web3torrent} from '../clients/web3torrent-client';
+import {useState} from 'react';
+import React from 'react';
+
+export const InitializationContext = React.createContext<
+  ReturnType<typeof useInitializationContext>
+>(undefined);
+export function useInitializationContext() {
+  const {paymentChannelClient} = web3torrent;
+
+  const [isInitialized, setInitialized] = useState(false);
```

Comment:
Should this be one state with 3 possible values (like an enum)? The values can be 'notInitialized', 'initializing', and ' initialized'.